appstore/dev_portal_api.py

```diff
@@ -42,6 +42,90 @@ def me():
     })
 
 
+@legacy_api.route('/users/me/developer', methods=['GET'])
+def my_apps():
+    result = authed_request('GET', f"{config['REBBLE_AUTH_URL']}/api/v1/me/pebble/appstore")
+    if result.status_code != 200:
+        abort(401)
+    me = result.json()
```

Comment:
This sequence happens 13 times in this PR. Consider making a helper function.

Reply:
Yes, I agree that it might be nice to have a `demand_auth_endpoint('/api/v1/me/pebble/appstore')` or similar...